README.md

```diff
@@ -1,32 +1,40 @@
-# StationAPI for Minecraft Beta 1.7.3 BIN Fabric
+# Minecraft Cursed Legacy API
 
-A general use API for Fabric Loader mods on legacy Minecraft versions.
+The (unofficial) [Fabric](https://fabricmc.net/) home for 1.2.5 and Beta 1.7.3 - if you want the newer versions see [here](https://github.com/FabricMC/fabric).
 
-## Plugin Setup
+## Setup
+Run the following command (if you are not using eclipse, replace “eclipse” with your relevant ide)
 
-Extra steps for better Mixin making and Fabric configuring in IntelliJ IDEA:
+```
+./gradlew eclipse
+```
 
-1. Go to `File > Settings...`
-2. Go to `Plugins` and install the `Minecraft Development` plugin.
-3. Restart IntelliJ IDEA.
-4. Follow the instructions in [Using This to Make Mods](#using-this-to-make-mods).
-5. Profit!
+NOTE: if you want sources (recommended), instead run this. You only need to run this the first time you need to generate sources.
 
-## Using This to Run Mods
+```
+./gradlew :rebuildLVT :genSources eclipse
+```
 
-You will want to install the [Cursed Fabric MultiMC Instance](https://github.com/calmilamsy/Cursed-Fabric-MultiMC) and [download the latest release of StationAPI](https://github.com/modificationstation/StationAPI/releases/latest).  
-Put the downloaded StationAPI jar into your mods folder, do NOT add as a jar mod.
+To be able to run the API as a whole in an API dev environment, ensure the root project has been set up in your ide. This should be automatic, but it pays to make sure.
 
-## Using This to Make Mods
+If you wish to build a copy of API, you can run:
 
-[See the wiki.](https://github.com/ModificationStation/StationAPI/wiki)
+```
+./gradlew build -x compileTestJava -x checkstyleTest
+```
 
-Proper code documentation coming soon. There are some JavaDoc comments for some commonly used classes such as ItemRegister and BlockRegister.
+## Contributing
 
-## Common Issues
+Make sure to follow the code style guidelines, which can be seen in use in existing files.
```

Comment:
Tell contributors of the Checkstyle-IDEA plugin which lets you import a checkstyle xml so that written code is automatically formatted